src/main/index.js

```diff
@@ -69,12 +69,15 @@ function createWindow () {
         },
         {
           label: 'Export to MIDI',
-          accelerator: 'CmdOrCtrl+E'
+          accelerator: 'CmdOrCtrl+E',
+          click: () => {
+            // TODO export midi file : Je ne sais pas comment accéder aux notes depuis ici
```

Comment:
Cf mon commentaire dans le corps de l'issue

Reply:
Super, je me doutais bien mais du coup faudrait qu'on enlève l'option export de la nav bar ?

Reply:
Oui, et puis pour l'instant je pense qu'on peut se contenter du bouton in-app